code/solr/tasks/Solr_BuildTask.php

```diff
@@ -22,7 +25,7 @@ class Solr_BuildTask extends BuildTask
      */
     public function getLogger()
     {
-        return Injector::inst()->get('Logger');
+        return Injector::inst()->get('Psr\Log\LoggerInterface');
```

Comment:
use `Injector::inst()->get(LoggerInterface::class)` instead. [This is mentioned in the changelog notes](https://docs.silverstripe.org/en/4/changelogs/4.0.0/)